src/interfaces/package-export-transport.ts

```diff
@@ -0,0 +1,20 @@
+import {StudioComputeNodeDescriptor} from "./package-manager.interfaces";
+
+export interface DependencyTransport {
```

Comment:
Are there any plans on using openapi  for generating these transports?

Reply:
I am not aware of any plans to use OpenApi at the moment.

Reply:
in that case I would suggest creating a ticket to have it documented as I would imagine that using openapi contract would be the ideal place to be for any client of the package-manager apis